src/app/containers/MediaPageMain/index.jsx

```diff
@@ -45,18 +32,55 @@ const MediaPageMain = props => {
       <main role="main">
         <Grid>
           <GridItemConstrainedMedium>
-            {blocks.map(renderBlock({ script, service }))}
-            <ul>
-              <li>
-                <strong>Service</strong>: {service}
-              </li>
-              <li>
-                <strong>Brand</strong>: {serviceId}
-              </li>
-              <li>
-                <strong>MediaId</strong>: {mediaId}
-              </li>
-            </ul>
+            {blocks.map(({ id, text, type, live, externalId }, index) => {
+              const isFirstBlock = index === 0;
+              const idAttribute = isFirstBlock ? SKIP_LINK_ANCHOR : null;
+              const blockType = live ? LIVE_RADIO_BLOCK : type;
+
+              switch (blockType) {
+                case HEADING_BLOCK:
+                  return (
+                    <Headline
+                      key={id}
+                      script={script}
+                      service={service}
+                      id={idAttribute}
+                    >
+                      {text}
+                    </Headline>
+                  );
+
+                case PARAGRAPH_BLOCK:
+                  return (
+                    <Paragraph
+                      key={id}
+                      script={script}
+                      service={service}
+                      id={idAttribute}
+                    >
+                      {text}
+                    </Paragraph>
+                  );
+
+                case LIVE_RADIO_BLOCK: {
+                  const MediaPlayer = {
+                    canonical: CanonicalMediaPlayer,
+                    amp: AmpMediaPlayer,
+                  }[platform];
+
+                  return (
+                    <MediaPlayer
+                      key={id}
+                      showPlaceholder={false}
+                      src={`/ws/av-embeds/media/${externalId}/${id}`}
```

Comment:
```suggestion
                      src={`/ws/av-embeds/media/${externalId}/liveradio`}
```

For this iteration we could go with something like